src/components/Directions.vue

```diff
@@ -1,12 +1,13 @@
 <template>
-  <ol v-if="this.directions.length">
-    <li :key='i' v-for="(direction, i) in directions">
+  <article v-if="this.directions.length">
+    <p class='direction' :key='i' v-for="(direction, i) in directions">
       {{direction}}  
```

Comment:
This is a good solution to our spacing issues for now. May be something to come back to later.